src/app/entities/courses/components/course-item/spec/host/course-item.component.spec.ts

```diff
@@ -36,6 +38,7 @@ describe('CourseItemComponent', () => {
       declarations: [
         CourseItemComponent,
         HostComponent,
+        ShowCourseStatusDirective,
```

Comment:
`CourseStatusDirective` ? or (if it is not tightly coupled with course) just `StatusDirective`

Reply:
I'll consider using of `AvailabilityStatusDirective`, as you've mentioned below